test/simple-render-tests.js

```diff
@@ -324,6 +346,32 @@ lab.experiment('simple render', () => {
     );
   });
 
+  lab.test('arrayOf custom type', ({ context }) => {
```

Comment:
Please remove this test, it's not testing the functionality of the `exact` type.